README
This is a school project from the secondary school Mariano José de Larra, Madrid.
We are attempting to improve a web page from a friend of our teacher (I think this might be child labor)
(Borja is sure this is child labor)(Marco knows this is child labor and is about to report it to the cops).

Write the changes down below
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓

v1.0
  -Starting version. (Complete shit)


changelog.md
# 0.0.0
### The Big Bang.
- Repository started.
- Contributors added.
- Readme file added.

_The project hasn't really started yet, we're waiting for the main files._

# 0.0.1
- Most original images are now in /assets.
- Folders for scripts and styles have been created.

_Main file finally here._

# 0.1.0
### Cosmic Expansion
> I (@kleber24021) have decided that enough advances have been made to jump in 0.1 in the version number :laughing:

- All dynamic parts of the main webpage are functional.
  - Header (although code is not pretty) works and looks cool.
  - Topnavbar is also working.
  - Sidenavbar works and looks pretty cool.
- New images (higher resolution) are now in /assets/a.
- Jquery has been installed and ~~surprisingly~~ it works.

_Protons and neutrons have started to join and form Hydrogen and Helium. Electrons joined in._

# 0.2.0
### Big Clouds of Gas
- Head is now a fully working carousel using Bootstrap.
- First JSON file containing content in Spanish has been created and structured.
- Flags have been added to the sidenavbar.
- Fonts have been changed.
- Content for "Galería" is mostly done.
- Content division is positioned.
- Added file "to-dos.md" and added tasks.
- main.js now uses JQuery syntax.
- Other, smaller or older changes.

_Atoms are slowly pulled together by their own mass. They could burst any moment now... Cosmic clouds await._


scripts/end.js
jQuery(document).ready(function ($) {

    var jssor_1_SlideshowTransitions = [
      {$Duration:800,$Opacity:2},
      {$Duration:700,$Opacity:2,$Brother:{$Duration:700,$Opacity:2}}
    ];

    var jssor_1_options = {
      $AutoPlay: 1,
      $Idle: 2500,
      $SlideDuration: 1000,
      $FillMode: 1,
      $SlideshowOptions: {
        $Class: $JssorSlideshowRunner$,
        $Transitions: jssor_1_SlideshowTransitions
      },
      $ArrowNavigatorOptions: {
        $Class: $JssorArrowNavigator$,
        $ChanceToShow: 1
      },
      $BulletNavigatorOptions: {
        $Class: $JssorBulletNavigator$,
        $ChanceToShow: 1
      }
    };

    var jssor_1_slider = new $JssorSlider$("jssor_1", jssor_1_options);

    /*#region responsive code begin*/

    var MAX_WIDTH = 1903;

    function ScaleSlider() {
        var containerElement = jssor_1_slider.$Elmt.parentNode;
        var containerWidth = containerElement.clientWidth;

        if (containerWidth) {

            var expectedWidth = Math.min(MAX_WIDTH || containerWidth, containerWidth);

            jssor_1_slider.$ScaleWidth(expectedWidth);
        }
        else {
            window.setTimeout(ScaleSlider, 30);
        }
    }

    ScaleSlider();

    $(window).bind("load", ScaleSlider);
    $(window).bind("resize", ScaleSlider);
    $(window).bind("orientationchange", ScaleSlider);
    /*#endregion responsive code end*/
});


scripts/main.js
function idfy(string) {
  var newstring = string.replace(/[^\w]/gi, '').toLowerCase();
  return newstring;
}

function setLang(lang) {
  let nav_lang = navigator.language.substr(0,2);
  if (lang) {
    page_lang = lang;
  }
  else if (nav_lang == 'en' || nav_lang == 'es' || nav_lang == 'fr') {
    page_lang = eval(nav_lang);
  }
  else {
    page_lang = en;
  }
}

function toggleSideNav(mode) {
  // Mode 0: toggle. Mode 1: close.
  let sideNavBar = $('#sidenavbar')[0];
  if (sideNavBar.style.left == '0px' || mode == 1) {
    sideNavBar.style.left = '-420px';
    for (var i = 0; i < 4; i++) {
      $('.content')[i].style.paddingLeft = '300px';
      $('.content')[i].style.paddingRight = '300px';
    }
    $('#menuicon')[0].classList = 'fas fa-bars';
  } else if (sideNavBar.style.left == '-420px' && mode != 1){
    sideNavBar.style.left = '0px';
    for (var i = 0; i < 4; i++) {
      $('.content')[i].style.paddingLeft = '510px';
      $('.content')[i].style.paddingRight = '90px';
    }
    $('#menuicon')[0].classList += 'fas fa-caret-left';
  }
}

function changeTab(tab_n) {
  let tab = page_lang['page']['structure'][tab_n]['tab_name'];
  let tab_c = page_lang['page']['structure'][tab_n]['tab_content'];
  let sidemenu_item_st = '<a href="#var1"><li class="sidenavmenuitem">var2</li></a>';
  active_content = '#content' + tab_n.toString();
  $('#sidenavmenu_u')[0].innerHTML = '';
  for (let i = 0; i < tab_c.length; i++) {
    $('#sidenavmenu_u')[0].innerHTML += sidemenu_item_st.replace(/var1/g, idfy(tab_c[i]['section_name'])).replace(/var2/g, tab_c[i]['section_name']);
  }
  for (let i = 0; i < 4; i++) {
    let cont_t = '#content' + i.toString();
    if (i < tab_n) {
      $(cont_t)[0].style.left = '-100%';
      $(cont_t)[0].style.opacity = '0';
    }
    else if (i > tab_n) {
      $(cont_t)[0].style.left = '100%';
      $(cont_t)[0].style.opacity = '0';
    }
    else {
      $(cont_t)[0].style.left = '0';
      $(cont_t)[0].style.opacity = '1';
    }
  }
}

function build_page() {
  let doc = page_lang['page'];
  $('#godownbt')[0].innerHTML = doc['godownbt_wait'];
  $('title')[0].innerHTML = doc['title'];
  $('.maintitle_a')[0].innerHTML = doc['maintitle_a'];
  for (let i = 0; i < $('.topnavmenuitem').length; i++) {
    $('.topnavmenuitem')[i].innerHTML = doc['structure'][i]['tab_name'];
  }
  for (var i = 0; i < $('.content').length; i++) {
    let cont_t = '#content' + i.toString();
    for (var j = 0; j < $(cont_t + '>.sect > .heading').length; j++) {
      let heading_text = doc['structure'][i]['tab_content'][j]['section_name'];
      $(cont_t + '> .sect')[j].id = idfy(heading_text);
      $(cont_t + '> .sect > .heading')[j].innerHTML = heading_text;
      let sect_id = '#' + idfy(heading_text);
       for (var k = 0; k < $(sect_id + '> .cpg').length; k++) {
         $(sect_id + '>.cpg')[k].innerHTML = doc['structure'][i]['tab_content'][j]['section_paragraphs'][k]
       }
    }
  }
  $('#godownbt')[0].innerHTML = doc['godownbt'] + '&nbsp; &nbsp;<i class="fas fa-angle-down"></i>';
  changeTab(0);
}

function scrolltopage(mode) {
  // Mode 0: Down, Mode 1: Up.
  var target = $('#topnavbar')[0];
  let targettop;
  mode == 1 ? targettop = -(target.offsetTop) : targettop = target.offsetTop;
  $('html, body').animate({scrollTop: targettop}, 1000, function() {
    if (mode == 1){
      target.focus();
    }
    else {
      $('#head')[0].focus();
    }});}


scripts/temp.js
const es = {
  'lang':'es',
  'page':{
    'title':'Las Jordanas - Casa Rural',
    'maintitle_a':'Casa Rural',
    'godownbt':'Explorar',
    'godownbt_wait':'Cargando',
    'backtopbt':'Volver al título',
    'logo':'';
    'structure':[
      {
        'tab_name':'Inicio',
        'tab_content':[
          {
            'section_name':'Bienvenidos',
            'section_paragraphs':[
              'La casa rural Las Jordanas se encuentra en el Parque Natural de Calblanque. A 2 km de las playas (unos 20 minutos a pie o 5 minutos en coche), y a 25 km de Cartagena; nuestra casa se halla en plena naturaleza, aislada, y a 2 km de Los Belones, el pueblo más cercano. Puede llegarse fácilmente desde las autovías A-30, AP-7 y RM-12, aunque si usan navegador les recomendamos o bien llamarnos al llegar a Los Belones, o bien consultar las instrucciones ofrecidas a continuación, ya que los navegadores indican erroneamente una vía en mal estado.'
            ],
          },
          {
            'section_name':'Nuestra historia',
            'section_paragraphs':[
              'Todo comenzó en 2010, después de años enamorados de esas playas salvajes, solitarias y de aguas limpias. Se cruzó en nuestra vida la casa de Las Jordanas, curiosa historia cuyos protagonistas fueron antiguos pobladores pertenecientes a la misma familia que habitó la casa ininterrumpidamente desde principios del siglo XIX: la Mañe, Los Hernández, el inmenso olivo del huerto, los palmitos, los granados... y que te contaremos con entusiasmo y cariño. La casa resolvía siempre nuestro problema cuando queríamos ir a Calblanque: ¿dónde vamos a dormir?¿En La Manga?¿En Cartagena?¿Cabo de Palos?¿Los Belones, quizás?... Se acabó: dormiríamos al fin escuchando al búho real por la noche, a los pájaros al despertar, viendo a las águilas sobrevolando el monte, oliendo el aroma del romero y del tomillo acariciados por la brisa del mar... junto a una de las mejores playas del Mediterráneo que aún quedan vírgenes.',
              'Tras un periodo de reformas que se nos hizo eterno, lleno de obstáculos y cómo no, una ingente cantidad de procesos burocráticos, ¡lo logramos! Podemos ahora disfrutar del Parque Natural de Calblanque en todas sus facetas y en todas las épocas del año. Pero también se ve cumplido este sueño para ti, querido viajero y amante de la naturaleza, de la tranquilidad, de lo distinto... pues este entorno privilegiado es ahora accesible para ti desde la finalización de este proyecto el 1 de junio de 2013, disfrutando esta maravillosa experiencia con nosotros.',
              'Estarás en un lugar perfecto donde descansar y disfrutar del Mare Nostrum. Encontrarás una casa del siglo XIX rehabilitada de la que hemos intentado respetar su estilo de casa de huerta murciana, levantina, mediterránea... Haciendo uso de materiales tradicionales y respetando al máximo todo lo que había en la casa antes de su rehabilitación, dispondrás en plena naturaleza de todas las comodidades, pudiendo elegir entre conectarte a Internet o desconectar y dejarte envolver por la naturaleza. Podrás pasear llegando a la cima del Cabezo de La Fuente y extasiarte con sus espectaculares vistas, recorrer la senda Circular al Cabezo de La Fuente, caminar a las playas de Calblanque, Salinas del Rasall, Negrete, Parreño, Dentoles, Las Mulas... Excursiones para todos los gustos. Podrás hacer uso además de nuestras bicicletas o montar a caballo si así lo prefieres. Si buscas unas vacaciones relajantes y no hacer mucho, nuestra casa será también tu rincón favorito. Podrás pasar agradables momentos en la antigua huerta, ya sea leyendo un libro o a la sombra del olivo de más de 300 años. Dejándote abrazar por los enormes palmitos, tomando el sol en las terrazas, echándote una siesta... ¡Te sentirás como en casa!'
            ],
          },
          {
            'section_name':'¿Por qué elegirnos?',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Galería',
        'tab_content':[
          {
            'section_name':'Casa',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Habitaciones',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Espacios Comunes',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Terraza',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Paisajes',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Información',
        'tab_content':[
          {
            'section_name':'Cómo Llegar',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Tarifas',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Tiempo',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Actividades',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Otros',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Contacto',
        'tab_content':[
          {
            'section_name':'Información de Contacto',
            'section_paragraphs':[
              'Puede contactarnos en: casalasjordanas@gmail.com o bien en +34 610 605 287 por Whatsapp o voz de 15h a 23h.'
            ],
          },
          {
            'section_name':'Síguenos en Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Créditos Página Web',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}


const en = {
  'lang':'en',
  'page':{
    'title':'Las Jordanas - Holiday Cottage',
    'maintitle_a':'Holiday Cottage',
    'godownbt':'Explore',
    'godownbt_wait':'Loading',
    'backtopbt':'Go back to the title',
    'logo':'';
    'structure':[
      {
        'tab_name':'Home',
        'tab_content':[
          {
            'section_name':'Welcome',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Our story',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Why pick us?',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Galery',
        'tab_content':[
          {
            'section_name':'House',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Bedrooms',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Shared spaces',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Garden and outside',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Views and landscapes',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Information',
        'tab_content':[
          {
            'section_name':'Accesing the house',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Prices',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Weather',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Activities',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Other',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Contact Us',
        'tab_content':[
          {
            'section_name':'Contact Info',
            'section_paragraphs':[
              'Our email is: casalasjordanas@gmail.com and our phone number is +34 610 605 287. Call us from 15h to 23h or text us through Whatsapp.'
            ],
          },
          {
            'section_name':'Follow us on Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Webmasters',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}


const fr = {
  'lang':'fr',
  'page':{
    'title':'Las Jordanas - Maison Rurale',
    'maintitle_a':'Maison Rurale',
    'godownbt':'Explorer',
    'godownbt_wait':'Chargement en cours',
    'backtopbt':'Retour au titre',
    'logo':'';
    'structure':[
      {
        'tab_name':'Accueil',
        'tab_content':[
          {
            'section_name':'Bienvenue',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Notre histoire',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Pourquoi nous choisir?',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Images',
        'tab_content':[
          {
            'section_name':'Notre maison',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Nos chambres',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Nos espaces communs',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Notre terrasse',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Paysages',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Infos pratiques',
        'tab_content':[
          {
            'section_name':'Comment y aller',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Tarif',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Météo',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Activités',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Autre',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Nous contacter',
        'tab_content':[
          {
            'section_name':'Information de contact',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Crédits de la page web',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}


scripts/build.js
const es = {
  'lang':'es',
  'page':{
    'title':'Las Jordanas - Casa Rural',
    'maintitle_a':'Casa Rural',
    'godownbt':'Explorar',
    'godownbt_wait':'Cargando',
    'backtopbt':'Volver al título',
    'logo':'',
    'structure':[
      {
        'tab_name':'Bienvenidos',
        'tab_content':[
          {
            'section_name':'Inicio',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Nuestra historia',
            'section_paragraphs':[
              'Todo comenzó en 2010, después de años enamorados de esas playas salvajes, solitarias y de aguas limpias. Se cruzó en nuestra vida la casa de Las Jordanas, curiosa historia cuyos protagonistas fueron antiguos pobladores pertenecientes a la misma familia que habitó la casa ininterrumpidamente desde principios del siglo XIX: la Mañe, Los Hernández, el inmenso olivo del huerto, los palmitos, los granados... y que te contaremos con entusiasmo y cariño. La casa resolvía siempre nuestro problema cuando queríamos ir a Calblanque: ¿dónde vamos a dormir?¿En La Manga?¿En Cartagena?¿Cabo de Palos?¿Los Belones, quizás?... Se acabó: dormiríamos al fin escuchando al búho real por la noche, a los pájaros al despertar, viendo a las águilas sobrevolando el monte, oliendo el aroma del romero y del tomillo acariciados por la brisa del mar... junto a una de las mejores playas del Mediterráneo que aún quedan vírgenes.',
              'Tras un periodo de reformas que se nos hizo eterno, lleno de obstáculos y cómo no, una ingente cantidad de procesos burocráticos, ¡lo logramos! Podemos ahora disfrutar del Parque Natural de Calblanque en todas sus facetas y en todas las épocas del año. Pero también se ve cumplido este sueño para ti, querido viajero y amante de la naturaleza, de la tranquilidad, de lo distinto... pues este entorno privilegiado es ahora accesible para ti desde la finalización de este proyecto el 1 de junio de 2013, disfrutando esta maravillosa experiencia con nosotros.',
              'Estarás en un lugar perfecto donde descansar y disfrutar del Mare Nostrum. Encontrarás una casa del siglo XIX rehabilitada de la que hemos intentado respetar su estilo de casa de huerta murciana, levantina, mediterránea... Haciendo uso de materiales tradicionales y respetando al máximo todo lo que había en la casa antes de su rehabilitación, dispondrás en plena naturaleza de todas las comodidades, pudiendo elegir entre conectarte a Internet o desconectar y dejarte envolver por la naturaleza. Podrás pasear llegando a la cima del Cabezo de La Fuente y extasiarte con sus espectaculares vistas, recorrer la senda Circular al Cabezo de La Fuente, caminar a las playas de Calblanque, Salinas del Rasall, Negrete, Parreño, Dentoles, Las Mulas... Excursiones para todos los gustos. Podrás hacer uso además de nuestras bicicletas o montar a caballo si así lo prefieres. Si buscas unas vacaciones relajantes y no hacer mucho, nuestra casa será también tu rincón favorito. Podrás pasar agradables momentos en la antigua huerta, ya sea leyendo un libro o a la sombra del olivo de más de 300 años. Dejándote abrazar por los enormes palmitos, tomando el sol en las terrazas, echándote una siesta... ¡Te sentirás como en casa!'
            ]
          },
          {
            'section_name':'¿Por qué elegirnos?',
            'section_paragraphs':[
              ''
            ]
          }
        ]
      },
      {
        'tab_name':'Galería',
        'tab_content':[
          {
            'section_name':'Casa',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Habitaciones',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Espacios Comunes',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Terraza',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Paisajes',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Información',
        'tab_content':[
          {
            'section_name':'Cómo Llegar',
            'section_paragraphs':[
              'La casa rural Las Jordanas se encuentra en el Parque Natural de Calblanque. A 2 km de las playas (unos 20 minutos a pie o 5 minutos en coche), y a 25 km de Cartagena; nuestra casa se halla en plena naturaleza, aislada, y a 2 km de Los Belones, el pueblo más cercano. Puede llegarse fácilmente desde las autovías A-30, AP-7 y RM-12, aunque si usan navegador les recomendamos o bien llamarnos al llegar a Los Belones, o bien consultar las instrucciones ofrecidas a continuación, ya que los navegadores indican erroneamente una vía en mal estado.'
            ],
          },
          {
            'section_name':'Tarifas',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Tiempo',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Actividades',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Otros',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Contacto',
        'tab_content':[
          {
            'section_name':'Información de Contacto',
            'section_paragraphs':[
              'Puede contactarnos en: casalasjordanas@gmail.com o bien en +34 610 605 287 por Whatsapp o voz de 15h a 23h.'
            ],
          },
          {
            'section_name':'Síguenos en Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Créditos Página Web',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}
const en = {
  'lang':'en',
  'page':{
    'title':'Las Jordanas - Holiday Cottage',
    'maintitle_a':'Holiday Cottage',
    'godownbt':'Explore',
    'godownbt_wait':'Loading',
    'backtopbt':'Go back to the title',
    'logo':'',
    'structure':[
      {
        'tab_name':'Home',
        'tab_content':[
          {
            'section_name':'Welcome',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Our story',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Why pick us?',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Galery',
        'tab_content':[
          {
            'section_name':'House',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Bedrooms',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Shared spaces',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Garden and outside',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Views and landscapes',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Information',
        'tab_content':[
          {
            'section_name':'Accesing the house',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Prices',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Weather',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Activities',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Other',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Contact Us',
        'tab_content':[
          {
            'section_name':'Contact Info',
            'section_paragraphs':[
              'Our email is: casalasjordanas@gmail.com and our phone number is +34 610 605 287. Call us from 15h to 23h or text us through Whatsapp.'
            ],
          },
          {
            'section_name':'Follow us on Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Webmasters',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}
const fr = {
  'lang':'fr',
  'page':{
    'title':'Las Jordanas - Maison Rurale',
    'maintitle_a':'Maison Rurale',
    'godownbt':'Explorer',
    'godownbt_wait':'Chargement en cours',
    'backtopbt':'Retour au titre',
    'logo':'',
    'structure':[
      {
        'tab_name':'Accueil',
        'tab_content':[
          {
            'section_name':'Bienvenue',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Notre histoire',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Pourquoi nous choisir?',
            'section_paragraphs':[
              ''
            ]
          }
        ]
      },
      {
        'tab_name':'Images',
        'tab_content':[
          {
            'section_name':'Notre maison',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Nos chambres',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Nos espaces communs',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Notre terrasse',
            'section_paragraphs':[
              ''
            ]
          },
          {
            'section_name':'Paysages',
            'section_paragraphs':[
              ''
            ]
          }
        ]
      },
      {
        'tab_name':'Infos pratiques',
        'tab_content':[
          {
            'section_name':'Comment y aller',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Tarif',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Météo',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Activités',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Autre',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      },
      {
        'tab_name':'Nous contacter',
        'tab_content':[
          {
            'section_name':'Information de contact',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Twitter',
            'section_paragraphs':[
              ''
            ],
          },
          {
            'section_name':'Crédits de la page web',
            'section_paragraphs':[
              ''
            ],
          },
        ]
      }
    ]
  }
}

let page_lang;
setLang();
build_page();
